### DISCUSSION TITLE
The Forest Server?

### DISCUSSION THREAD

**Discussion by HerrDummkopf** [GameServerManagers/LinuxGSM#3698] [Q&A] [answered]
do you pla a The Forest Server ?

**Comment by Claiyc** [comment, marked as answer]
The forest does not seem to have linux binaries. We do not support wine! So no, the forest server won't be added until they release linux binaries.

**Comment by OdinVex** [comment]
Steam is offering the download for Linux users through Proton. Perhaps that would be alright?

**Reply by Claiyc** [reply]
we do not support proton, wine or similar
